### DISCUSSION TITLE
hello

### DISCUSSION THREAD

**Discussion by samerhello** [community/community#128234] [Programming Help]
### Select Topic Area

Question

### Body

hello i am using chesspartner software to import uci stockfish engine but i noticed that new error occures , how to show that

**Comment by samerhello** [comment]
and the error begins from one development version

**Comment by samerhello** [comment]
how to get answer

**Comment by samerhello** [comment]
HELLO
You can go to ( https://github.com/official-stockfish/Stockfish/issues/5393
) to see especially attached gif

**Comment by sandipanrakshit34** [comment]
hello
